### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Oluwadamilareolusakin**
We styled the elements on the first and second section of the smash magazine website with background colors from really dark to light. The darkest colors signify elements with the highest visual hierarchy and the lightest signifies the one will the littlest.

### CODE REVIEW

**Comment by samgaco** [comment]
Hi @Oluwadamilareolusakin and @CornetS28  :hand: 

Everything is going in the right direction :+1: . Some things to work on:

1. You should at least add images (when they appear in the original website) in the top left corner of the articles. The animation is not needed, but an embedded image should be there (it doesn't need to be the same image).
article

2. Bear in mind the following semantic structure; All content should be wrapped within` <main>` tags (you only have a small portion of the content currently inside the `<main>` tags), every child of main should be enclosed within a `<section>` tag and every child of section within an `<article>` tag.

3. You didn't implement some sections and different design articles such as:

* This red articles here:
sec

* This other article area at the end:
sec2

* This section right before the footer:
sec3

* In general, you should think of this project as if it were an outline of the original, **you don't need to add everything exactly as in the original, but it should feel like it does have the structure and design of the original.**

4. Add a project description to your readme file, it is also suggested to add your contacts, names and a live demo of the site.

The rest looks perfect so far, please :pray:  work on these points and submit back for a code review. Regards! :muscle:

**Comment by Ispirett** [comment]
Hey folks great work so far on this project :100:  There some changes that need to addressed for this project to be completed.
- [ ]  **Navigation bar**
   It should use the <nav> HTML tag
 It should have seven separate containers (<div> is recommended)
- [ ] In your lates post section the articles show be wrapped in `<article>` tag

**Main content**

The main content should be made from separate containers, stacked on top of each other and in the following arrangement:

- [ ] All content should be wrapped with `<main>` HTML tags
- [ ] Every child of main should be a `<section>` HTML tag pair
- [ ] Every child of section should be an `<article>` HTML tag pair

File: index.html [hunk 1]
```diff
@@ -0,0 +1,350 @@
...
+                        <p class="first-line flex-row">5 DAYS AGO</p>
+                        <h1 class="second-line">UX Improvements For <br> Keyboard Accessibility</h1>
+                        <h3 class="third-line">by Vitaly Friedman</h3>
+                        <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Eaque tempora c
+                            orrupti ab architecto veniam minus, a similique autem odio? Architecto, eum. 
+                            Nam tempore obcaecati saepe adipisci porro necessitatibus quibusdam quaerat.</p>
+                    </div>
+                </div>
+             </main>
+    <footer class="footer">
```
Comment (Ispirett) [inline index.html]: Please use `list-style:none' for li tags. Also, you can try and organize the links a little more like the site

File: index.html [hunk 2]
```diff
@@ -0,0 +1,350 @@
...
+            
+                    <div class="post banner">
+                        <p class="picture"></p>
+                        <h2 class="banner-header">1,499 smashing members</h2>
+                        <p class="banner-content"></p>
+                        <p class="banner-sign-in"></p>
+                    </div>
+               </div>
+
+               <div id="latest-posts" class="flex-row">
```
Comment (Ispirett) [inline index.html]: Please use classes instead of ids. You have duplicated ids

File: index.html (hunk 1 above)
Comment (Oluwadamilareolusakin) [reply index.html]: Noted with thanks

File: index.html (hunk 2 above)
Comment (Oluwadamilareolusakin) [reply index.html]: Noted with thanks

**Comment by Oghenebrume50** [comment]
Good work so far team, all corrections were implemented properly, what is left now is to make the site responsive.
So for big desktop screens - u can have large three articles display in one horizontal line
For slightly smaller screens, you can have 2 articles displays in one horizontal line
For small screens like mobile, one article per horizontal line, so each article should span the full width of the container
Is that clear enough? Whatever size you choose for your container width does not matter, just make sure it is responsive.
A good way to understand this is to slowly minimize the original site and watch how each element's position changes, please mimic that, to the best of your ability.

All these can be done, easily with flexbox. Please fix that and submit for re-review.

**Comment by bolah2009** [comment]
Currently on review

**Pull request merged**